docs/contributing/gsoc.md

```diff
@@ -424,7 +424,7 @@ CSS](https://github.com/zulip/zulip/).
   settings/administration/streams overlays in a development
   environment. You can get experience working on the subsystem by
   working on some of [our open settings/admin
-  issues](https://github.com/zulip/zulip/labels/area%3A%20admin). 175
+  issues](https://github.com/zulip/zulip/issues?q=is%3Aopen+is%3Aissue+label%3A%22area%3A+settings+%28admin%2Forg%29%22%2C%22area%3A+settings+%28user%29%22%2C%22area%3A+stream+settings%22). 175
```

Comment:
Let's use a Markdown reference to put the URL on its own line for readability.

Also, should "settings UI" be in the list of labels we filter ofr?

Reply:
@alya mentioned these 3 labels... but she also mentioned that the intention is to filter _any_ of the settings issues; so I think the "settings UI" label should also be included.

Reply:
I'm waiting for a confirmation on this. For the time being, I have included the "Settings UI" label as well.